pkg/cmd/openshift-controller-manager/cmd.go

```diff
@@ -64,6 +65,7 @@ func NewOpenShiftControllerManagerCommand(name string, out, errout io.Writer, ct
 	flags := cmd.Flags()
 	// This command only supports reading from config
 	flags.StringVar(&options.ConfigFilePath, "config", options.ConfigFilePath, "Location of the master configuration file to run from.")
+	flags.StringVar(&options.KubeConfigFilePath, "kubeconfig", options.ConfigFilePath, "This flag is ignored.")
```

Comment:
```suggestion
	flags.StringVar(&options.KubeConfigFilePath, "kubeconfig", options.KubeConfigFilePath, "This flag is ignored.")
```

even though it is temporary, let's not mix up the defaults

Reply:
Amended. I actually also needed to add `--namespace` in the end.